doc/cookbook/source/examples/classifier/multiclass_logisticregression.rst

```diff
@@ -0,0 +1,49 @@
+===============================
+Multi-class Logistic Regression
+===============================
+
+Multinomial logistic regression assigns the sample :math:`\mathbf{x}_i` to class :math:`c`
+based on :math:`P`, the probability for sample :math:`\mathbf{x}_i`: to be in class :math:`c`:
+
+.. math::
+
+    P(Y_i = c | \mathbf{X}_i) = \frac{\mathrm{exp}(\mathbf{\theta}^T_c\mathbf{X}_i)}{1+ \sum_{k=1}^{K}\mathrm{exp}(\mathbf{\theta}^T_k\mathbf{X}_i)}
```

Comment:
btw if you locally run `make cookbook` then you can look at the rendered html output (See readme).
If you do this, you can spot latex errors in this one (it is not rendered properly), as also visible [here](http://buildbot.shogun-toolbox.org/cookbook_pr/583fd0d3f73ac5166e680e4a7f574aed41ee688b/examples/classifier/multiclass_logisticregression.html)